document-service-api/src/lib/config.js

```diff
@@ -7,6 +7,10 @@
  */
 
 module.exports = {
+  AZURE_BLOB_STORAGE_CONNECTION_STRING: process.env.AZURE_BLOB_STORAGE_CONNECTION_STRING,
```

Comment:
I don't like the capitalised keys in the config. I would also suggest putting it inside an object for consistency. Something like:

```
blobStorage: {
  connection: process.env.AZURE_BLOB_STORAGE_CONNECTION_STRING
}
```